orchestrator-salt/src/main/resources/salt/pillar/datalake_resize/check_atlas_updated.sls

```diff
@@ -0,0 +1,2 @@
+check_atlas_updated:
+  max_retries:
```

Comment:
could you remove empty pillars? and ensure the top file has the necessary if statement

Reply:
The idea is that the variable `max_retries` will be populated either manually or by an automated process. I have seen other pillar files do this and mimicked them since we don't want a hardcoded value here.

Also what do you mean by the if statement? Do you want me to check if the script file actually exists?

Reply:
we have removed empty pillars to avoid such case for salt upgrade flow fails midway and leaves behind empty pillars

Reply:
also I don't see where is this populated

Reply:
Hmm I see, well the idea is that the value for `max_retries` will be populated in one of two ways: manually by a user within this file itself on the DL node, and via a CB Salt orchestrator command. The CB logic will be part of a future PR.

Do you want me to just have a default value for `max_retries` for now? I don't mind doing that.

Reply:
Understood, for now I will add a default value since we will use this file in the future when making control plane overwrites. Thanks for the advice Mihaly.